### PR TITLE
Add shortcut to mute sound

### PR DESCRIPTION
**Pull request by runelaenen**
As asked for here: https://openrct2.org/forums/topic/976-muting-sound/

### CODE REVIEW

File: src/config.c [hunk 1]
```diff
@@ -1024,6 +1024,7 @@ static const uint16 _defaultShortcutKeys[SHORTCUT_COUNT] = {
 	PLATFORM_MODIFIER | SDL_SCANCODE_F10,	// SHORTCUT_QUICK_SAVE_GAME
 
 	SHORTCUT_UNDEFINED,					// SHORTCUT_SHOW_OPTIONS
+	SHORTCUT_UNDEFINED,					// SHORTCUT_SHOW_OPTIONS
```
Comment (Gymnasiast) [inline src/config.c]: Comment is incorrect (copy-pasted).

**Comment by IntelOrca** [comment]
Unable to merge, this isn't committed after the latest develop.

**Comment by Gymnasiast** [comment]
@IntelOrca Would
```
gConfigSound.sound_enabled = !gConfigSound.sound_enabled;
```
be better, then?

This needs rebasing as well.

**Comment by IntelOrca** [comment]
@Gymnasiast for now yeah, it will just mean that that persists throughout the game and future sessions.

**Comment by Gymnasiast** [comment]
@IntelOrca I don't think that is bad per se, as long as it is not too easy to accidentally mute the sound.

File: src/windows/options.c [hunk 2]
```diff
@@ -628,7 +628,7 @@ static void window_options_mouseup(rct_window *w, int widgetIndex)
 	case WINDOW_OPTIONS_PAGE_AUDIO:
 		switch (widgetIndex) {
 		case WIDX_SOUND_CHECKBOX:
-			// audio_toggle_all_sounds();
+			//audio_toggle_all_sounds();
```
Comment (Gymnasiast) [inline src/windows/options.c]: You can get rid of this line altogether. We have source control for that. :-)

**Comment by IntelOrca** [comment]
@Gymnasiast would you like to take ownership of this PR?

**Pull request closed without merging**